src/helpers.rs

```diff
@@ -462,7 +462,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         let target_os = self.eval_context_ref().tcx.sess.target.target.target_os.as_str();
         match target_os {
             "linux" | "macos" => self.read_os_str_from_c_str(scalar).map(|x| x.to_os_string()),
-            "windows" => self.read_os_str_from_wide_str(scalar),
+            "windows" => { self.read_os_str_from_wide_str(scalar) },
```

Comment:
What are the curly braces for?

Reply:
Oh I forgot to delete the curly braces.. Please blame me for my clumsiness..